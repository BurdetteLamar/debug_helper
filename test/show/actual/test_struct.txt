---
DebugHelperTest::MyStruct (size=3 name=test_struct):
  Member 0:
    Name: :foo
    Value: 0 (Fixnum)
  Member 1:
    Name: :bar
    Value: 1 (Fixnum)
  Member 2:
    Name: :baz
    Value: 2 (Fixnum)
