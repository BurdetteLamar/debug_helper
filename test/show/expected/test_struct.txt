---
DebugHelperTest::MyStruct (message='test_struct' size=3):
  Member 0:
    Name: :foo
    Value: Fixnum 0
  Member 1:
    Name: :bar
    Value: Fixnum 1
  Member 2:
    Name: :baz
    Value: Fixnum 2
